examples/v2/matterbridge/config_chat2bridge.nim

```diff
@@ -66,14 +66,25 @@ type
       name: "metrics-server-port" .}: uint16
 
     ### Waku v2 options
-    staticnodes* {.
+    # @TODO: deprecate this item. Name changed from `staticnode` -> `staticnodes`
+    staticnodes_depr* {.
       desc: "Multiaddr of peer to directly connect with. Argument may be repeated"
       name: "staticnode" }: seq[string]
+    
+    staticnodes* {.
+      desc: "Multiaddr of peer to directly connect with. Argument may be repeated"
+      name: "staticnodes" }: seq[string]
```

Comment:
I use this singular typically because of the way it works in confutils. At CLI it expects the usage as such `--staticnode:... --staticnode:...` for multiple staticnodes. 
The actual variable can be plural as it holds the full provided list in a `seq`. 
So to me it makes more sense like this from a UX PoV. It would be different when the CLI would expect a comma separated list. I've seen some users making that error in the past actually, trying to provide the comma separated list when it was called `--staticnodes`

Reply:
This is a good point. As long as `--help` clearly indicates that it can be used multiple times.

It would also match how flags like these are named in Nimbus:
https://github.com/status-im/nimbus-eth2/blob/e1a8049e/beacon_chain/conf.nim#L109-L111
https://github.com/status-im/nimbus-eth2/blob/e1a8049e/beacon_chain/conf.nim#L314-L316